doc/User_profile.json

```diff
@@ -79,56 +79,35 @@
 	],
 	"picture": "https://s.gravatar.com/avatar/...png",
 	"shirtSize": "L",
-	"membership": {
-		"Workday": [
+	"groups": [
```

Comment:
yes, authoritativegroups need extra attributes that regular groups dont care for
the authoritative groups would be created by auth0 or directly by CIS (ie adding a new authoritative group that is mozillians-managed is manual)